implementations/rust/ockam/ockam_transport_udp/tests/tests.rs

```diff
@@ -0,0 +1,250 @@
+use ockam::errcode::Origin;
+use ockam::{Address, Error};
+use ockam_core::compat::rand::{self, Rng};
+use ockam_core::{route, AllowAll, Result, Routed, Worker};
+use ockam_node::Context;
+use ockam_transport_udp::{UdpTransport, UDP};
+use std::net::SocketAddr;
+use std::time::Duration;
+use tokio::net::UdpSocket;
+use tracing::{debug, error, trace};
+
+const TIMEOUT: Duration = Duration::from_secs(5);
+const AVAILABLE_LOCAL_PORTS_ADDR: &str = "0.0.0.0:0";
```

Comment:
Can you bind to 127.0.0.1 rather than 0.0.0.0?

Reply:
Okay.